support/src/traits.rs

```diff
@@ -75,3 +75,8 @@ pub trait GenerateBenchmarkOrigin<OuterOrigin, AccountId, SubjectId> {
 pub trait GetWorstCase {
 	fn worst_case() -> Self;
 }
+
+/// Generic filter.
+pub trait ItemFilter<Item> {
+	fn should_include(&self, credential: &Item) -> bool;
+}
```

Comment:
there is a similar trait in Frame support(?).

Reply:
Are you talking about [InstanceFilter](https://paritytech.github.io/substrate/master/frame_support/traits/trait.InstanceFilter.html) ?

Reply:
Yes, but in the last conversation we had (https://github.com/KILTprotocol/kilt-node/pull/378#discussion_r958170843)
we agreed we don't need any superset logic, so I would rather not use that. I would be happy to use any other trait that would work, otherwise we can keep it as it is, and get rid of it in the future if needed.